### ISSUE TITLE
Leverage bakthat library

### ISSUE THREAD

**Issue by pezon** [mittsh/db-s3-backup#7]
Leverage the bakthat library to implement backing up various databases or file descriptors,using bakthat to deal with the hairy issues, such as:

* compression
* encryption
* local backups
* backup rotations

http://bakthat.readthedocs.org/en/latest/

**Comment by mittsh** [comment]
@pezon yeah that's a great idea, I didn't know [bakthat](http://bakthat.readthedocs.org/en/latest/) but it looks great. Though I'm not working on this project anymore, so I won't be able to do it myself... PRs welcome of course :smile:
